src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -1107,6 +1113,11 @@ private void UpdateNextRefreshTime(KeyValueWatcher changeWatcher)
                 }
 
                 previousEndpoint = currentEndpoint;
```

Comment:
This comment really refers to the line 1110:

In what situation it reach out here with `previousEndpoint == currentEndpoint`?

Reply:
Some corner cases like if you set some identical connection strings, then there will be duplicated endpoints in the client list.

Reply:
Okay, then do we need to put the  `_requestTracingOptions.IsFailoverRequest = true;` into this `if (previousEndpoint != currentEndpoint)` block?

Reply:
I prefer to not put it into `if (previousEndpoint != currentEndpoint)` block. I think the current way is easier to read for me.

Reply:
It affects the result, when `previousendpoint == currentEndpoint` it still be treated as failover, a little bit odd, but I agree that it's not a big deal if it is just a super corner case.